samples/ChatSample/ChatSample/appsettings.json

```diff
@@ -1,7 +1,14 @@
 ﻿{
   "Azure": {
     "SignalR": {
-      "ConnectionString": ""
+      "ConnectionString": [
+        {
+          "EP1:Primary": ""
```

Comment:
revert?

Reply:
sample to show how to configure multiple endpoints through ConnectionString.

---
In reply to: [374090429](https://github.com/Azure/azure-signalr/pull/801#discussion_r374090429) [](ancestors = 374090429)